BD-3358 Apply templates to non-nested data & analytics docs (#7865)
braze-inc/braze-docs: _docs/_user_guide/data_and_analytics/report_metrics.md

Content Cards, Email, In-App Message, Web Push, iOS Push, Android Push, Webhook, SMS
{% endapitags %}

> Additional conversion events added after the Primary Conversion Event. The number of times a defined event occurred after interacting with or viewing a received message from a Braze campaign. <br><br> This defined event is determined by the marketer when building the campaign. For Email, Push and Webhooks, we start tracking conversions after the initial send. For Content Cards and In-App Messages, this count begins when they view a Content Card or Message for the first time.

Reviewer: ```suggestion
Additional conversion events added after the primary conversion event. The number of times a defined event occurred after interacting with or viewing a received message from a Braze campaign. <br><br> This defined event is determined by the marketer when building the campaign. For email, push, and webhooks, we start tracking conversions after the initial send. For Content Cards and in-app messages, this count begins when they view a Content Card or message for the first time.
```

SMS
{% endapitags %}

> The SMS has been rejected by the carrier. This can happen for a number of reasons, including carrier content filtering, availability of the destination device, the phone number is no longer in service, and similar. As a Braze customer, rejections are charged toward your SMS allotment.

Reviewer: ```suggestion
The SMS has been rejected by the carrier. This can happen for several reasons, including carrier content filtering, availability of the destination device, the phone number is no longer in service, and similar. As a Braze customer, rejections are charged toward your SMS allotment.
```

*Sends to Carrier* is deprecated, but will continue to be supported for users that already have it.
{% endalert %}

> The sum of Confirmed Deliveries, Rejections, and Sends where delivery or rejection wasn't confirmed by the carrier. This includes instances where carriers don't provide delivery or rejected confirmation, as some carriers don't provide this confirmation or can't do so at the time of send.

Reviewer: ```suggestion
The sum of *Confirmed Deliveries*, *Rejections*, and *Sends* where delivery or rejection wasn't confirmed by the carrier. This includes instances where carriers don't provide delivery or reject confirmation, as some carriers don't provide this confirmation or can't do so at the time of sending.
```

Email, LINE
{% endapitags %}

> This is the total number of delivered emails that have been opened by a single user at least once, and is tracked over a 7-day period. For LINE, this is tracked after a minimum threshold of 20 messages per day has been reached.

Reviewer: ```suggestion
This is the total number of delivered emails that have been opened by a single user at least once and are tracked over a 7-day period. For LINE, this is tracked after a minimum threshold of 20 messages per day has been reached.
```
